CHANGELOG.md

```diff
@@ -1,5 +1,17 @@
 # Changelog
 
+## v1.0.0
+
+### Added
+
+- Added an optional callback per job which defines what to do when that job errors.
```

Comment:
Did you also change the way it saves errors?

Reply:
No I took that out